app/locales/en.json

```diff
@@ -155,11 +155,11 @@
   "url": "Target URL",
   "tag": "Tag",
 
-  "my_accounts title": "My accounts",
-  "my_accounts category title": "Categories",
-  "my_accounts discovery title": "Discovery",
-  "my_accounts pinned title": "Pinned",
-  "my_accounts coming_soon": "Coming soon!",
+  "data_connect title": "Data Connect",
```

Comment:
As reminded by @CPatchane, let's remove all references to app name in translations keys.